diff --git a/filters/user_filters.c b/filters/user_filters.c
index 7bdda6965..5d83b92ea 100644
--- a/filters/user_filters.c
+++ b/filters/user_filters.c
@@ -83,5 +83,6 @@
 const struct mp_user_filter_entry *vf_list[] = {
     &vf_format,
+    &vf_rife_ncnn,
     &vf_lavfi,
     &vf_lavfi_bridge,
     &vf_sub,
diff --git a/filters/user_filters.h b/filters/user_filters.h
index aa88649b3..54669440a 100644
--- a/filters/user_filters.h
+++ b/filters/user_filters.h
@@ -38,3 +38,4 @@ extern const struct mp_user_filter_entry vf_d3d11vpp;
 extern const struct mp_user_filter_entry vf_amf_frc;
 extern const struct mp_user_filter_entry vf_fingerprint;
 extern const struct mp_user_filter_entry vf_gpu;
+extern const struct mp_user_filter_entry vf_rife_ncnn;
diff --git a/meson.build b/meson.build
index 5ffe55976..4e8b70628 100644
--- a/meson.build
+++ b/meson.build
@@ -212,6 +212,7 @@ sources = files(
     'video/decode/vd_lavc.c',
     'video/filter/refqueue.c',
     'video/filter/vf_format.c',
+    'video/filter/vf_rife_ncnn.c',
     'video/filter/vf_sub.c',
     'video/fmt-conversion.c',
     'video/hwdec.c',
